README.md

```diff
@@ -75,6 +75,11 @@ the binary verification command above.
 
 ## Build
 
+To quickly test that io.js was built correctly from any of the steps below:
```

Comment:
I don't think it makes sense to explain how to test the build before building. I'd move this to the end of the build instructions.